### DISCUSSION TITLE
Add an Expressive Text-to-Speech Feature

### DISCUSSION THREAD

**Discussion by Janvierscode** [dailydotdev/daily#1242] [Feature Requests]
# Hey Daily.dev team! 👋

I love the content on Daily.dev, but sometimes I wish I could consume it hands-free. 🤔 Would it be possible to add a **'Read Aloud' feature** with an expressive, human-like voice, similar to the **Expressive Text-to-Speech** used in advanced AI models? 🤖

Imagine being able to listen to articles, news, and tutorials while **commuting**, **exercising**, or just **taking a break**! 🎧 What do you think, fellow devs? Would you use this feature? 🤔

**Let's discuss! 💬**

**Comment by Janvierscode** [comment]
As a developer, I often find myself multitasking and on the go. Having a 'Read Aloud' feature with expressive, human-like voice capabilities would be incredibly beneficial. It would allow me to keep up with the latest articles, news, and tutorials even when I can't actively read them. This feature could enhance accessibility and convenience, making the rich content on Daily.dev more versatile and user-friendly. I believe many others in the community would also appreciate and use this feature regularly.

**Comment by idoshamun** [comment]
Hi @Janvierscode,

Thanks a lot for the suggestion. We will consider it. It's an interesting one
